coding-concepts.js

```diff
@@ -16,44 +16,44 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It will log "9" because it's inside of a string it will individually count each character. 
+// b) Verify and explain: The answer was 10, which means I believe it counted the space in between?
```

Comment:
Correct!  A string is a collection of characters, and even a space is considered a character.